ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh

```diff
@@ -119,10 +130,12 @@ set +e
 for i in "${!DATACENTERS[@]}"; do
   echo "$(date -u --rfc-3339=seconds) - Find virtual machines attached to ${vsphere_portgroup} in DC ${DATACENTERS[$i]} and destroy"
   DATACENTER=$(echo -n ${DATACENTERS[$i]} | tr -d '\n')
-  govc ls -json "/${DATACENTER}/network/${vsphere_portgroup}" |
-    jq '.elements[]?.Object.Vm[]?.Value' |
-    xargs -I {} --no-run-if-empty govc ls -json -L VirtualMachine:{} |
-    jq '.elements[].Path | select((contains("ova") or test("\\bci-segment-[0-9]?[0-9]?[0-9]-bastion\\b")) | not)' |
-    xargs -I {} --no-run-if-empty govc vm.destroy {}
+  for portgroup in "${portgroup_list[@]}"; do
+     govc ls -json "/${DATACENTER}/network/${portgroup}" |
+      jq '.elements[]?.Object.Vm[]?.Value' |
+      xargs -I {} --no-run-if-empty govc ls -json -L VirtualMachine:{} |
+      jq '.elements[].Path | select((contains("ova") or test("\\bci-segment-[0-9]?[0-9]?[0-9]-bastion\\b")) | not)' |
```

Comment:
Per @jinyunma 's suggestion, I guess here it should be updated to:
```
jq '.elements[].Path | select( ( contains("ova") ) | not)' |
```

Reply:
I double checked on IBMC vsphere7, here vm with suffix "bastion" maybe refer to below bastion host, should not the one QE created for disconnected network.
```
$ govc find -type m | grep bastion
/IBMCloud/vm/bastion
/IBMCloud/vm/jcallen-bastion
/IBMCloud/vm/jcallen-temp-bastion
/IBMCloud/vm/ngirard-dev-bastion
/IBMCloud/vm/rvanderp-ci-dev-bastion
/IBMCloud/vm/ci-segment-94-bastion
/IBMCloud/vm/ci-segment-89-bastion
/IBMCloud/vm/ci-segment-106-bastion
/IBMCloud/vm/ci-segment-109-bastion
/IBMCloud/vm/ci-segment-90-bastion
/IBMCloud/vm/ci-segment-88-bastion
/IBMCloud/vm/rhcos-412.86.202212081411-0-vmware.x86_64.ova-bastion
/IBMCloud/vm/ci-segment-102-bastion
/IBMCloud/vm/ci-segment-108-bastion
/IBMCloud/vm/ci-segment-92-bastion
/IBMCloud/vm/ci-segment-105-bastion
/IBMCloud/vm/ci-segment-99-bastion
/IBMCloud/vm/ci-segment-91-bastion
/IBMCloud/vm/ci-segment-97-bastion
/IBMCloud/vm/ci-segment-107-bastion
/IBMCloud/vm/ci-segment-93-bastion
/IBMCloud/vm/dev-ci-workload-1-bastion
/IBMCloud/vm/ci-segment-103-bastion
/IBMCloud/vm/ci-segment-95-bastion
/IBMCloud/vm/ci-segment-111-bastion
/IBMCloud/vm/ci-segment-104-bastion
/IBMCloud/vm/ci-segment-100-bastion
/IBMCloud/vm/ci-segment-101-bastion
/IBMCloud/vm/ci-segment-110-bastion
/IBMCloud/vm/ci-segment-98-bastion
/IBMCloud/vm/ci-segment-96-bastion
```
sorry for incorrect suggestion, @WenXinWei could you please add it back? thanks.

Reply:
ok,got it.